Makefile

```diff
@@ -13,12 +13,10 @@ test: $(source) $(tests)
 
 docs: docs/index.html
 docs/index.html: $(source)
-	pdoc ./arxiv -o docs
+	pdoc ./arxiv/arxiv.py -o docs
 	mv docs/arxiv/arxiv.html docs/index.html
-	rm docs/arxiv.html
 	rmdir docs/arxiv
-	# Fixup relative path.
-	sed -i.bak 's/\.\./\./g' docs/index.html; rm docs/*.bak
```

Comment:
One note: it *is* nice to have fuzzy search in the sidebar, even for a package this simple. I'll keep an eye on your changelog in case you re-add it!

Reply:
Yeah, I see where you are coming from. The intention here was that users are already familar with their browser's builtin search, and not showing the in-page search makes it clear that there is nothing else that can be searched. :) I'll gather a bit more feedback, if it turns out that folks universally still want fuzzy search, we may just re-add it. :)